scripts/tools/PropagateEmptyResponseVariables.php

```diff
@@ -94,22 +96,42 @@ protected function run()
         $variables = $this->createVariables();
         foreach ($deliveryExecutionIdList as $deliveryExecutionId) {
             $deliveryExecution = $this->getServiceProxy()->getDeliveryExecution($deliveryExecutionId);
-            if (in_array($deliveryExecution, self::DE_ENDED_STATUSES, true)) {
-                $outcome[] = sprintf('[%s] Delivery execution not finished', $deliveryExecutionId);
+            if (!in_array($deliveryExecution->getState()->getUri(), self::DE_ENDED_STATUSES, true)) {
```

Comment:
This means this if always passed yesterday? 

```
if (in_array($deliveryExecution, self::DE_ENDED_STATUSES, true)) {
```

Reply:
but for our case with deliveries if they have `test variables` in result they should be finished, so we good and i check those 8 which not passed they are terminated
so we didn't affect anything unexpected
but yes check was broken :(

Reply:
Yes, we only executed to isolated Delivery Executions that did not have any response before.